### DISCUSSION TITLE
When ImageMagick7.0.8 to 7.0.11 version up  coders/heic.c [-Wunused-variable]

### DISCUSSION THREAD

**Discussion by gold-castle** [ImageMagick/ImageMagick#3885] [Help]
When upgrading with Image Magic, the following warning message will appear.
Please tell me how to deal with it.

Below is the server information.
OS RHEL 7.5
libheif-1.3.2.-2.el7.x86_64

warning message-----------------------------
coders/heic.c:関数’ReadHEICColorProfile' 内
coders/heic.c142:5：警告：使用されない変数'length'です。[-Wunused-variable]
length

-----------------------------

The relevant part of heic.c is as follows, but is it okay to ignore the warning?

-----------------------------

size_t
    length;

  /*
    Read color profile.
  */
#if LIBHEIF_NUMERIC_VERSION >= 0x01040000
  length=heif_image_handle_get_raw_color_profile_size(image_handle);
  
-----------------------------

**Comment by dlemstra** [comment]
Thanks for reporting this, this warning will be gone in the next release.
